.travis.yml

```diff
@@ -39,43 +40,6 @@ env:
   - ANSIBLE_VERSION=2.7.2
   - ANSIBLE_VERSION=2.7.1
   - ANSIBLE_VERSION=2.7.0
-  - ANSIBLE_VERSION=2.6.20
-  - ANSIBLE_VERSION=2.6.19
-  - ANSIBLE_VERSION=2.6.18
-  - ANSIBLE_VERSION=2.6.17
-  - ANSIBLE_VERSION=2.6.16
-  - ANSIBLE_VERSION=2.6.15
-  - ANSIBLE_VERSION=2.6.14
-  - ANSIBLE_VERSION=2.6.13
-  - ANSIBLE_VERSION=2.6.12
-  - ANSIBLE_VERSION=2.6.11
-  - ANSIBLE_VERSION=2.6.10
-  - ANSIBLE_VERSION=2.6.9
-  - ANSIBLE_VERSION=2.6.8
-  - ANSIBLE_VERSION=2.6.7
-  - ANSIBLE_VERSION=2.6.6
-  - ANSIBLE_VERSION=2.6.5
-  - ANSIBLE_VERSION=2.6.4
-  - ANSIBLE_VERSION=2.6.3
-  - ANSIBLE_VERSION=2.6.2
-  - ANSIBLE_VERSION=2.6.1
-  - ANSIBLE_VERSION=2.6.0
-  - ANSIBLE_VERSION=2.5.15
-  - ANSIBLE_VERSION=2.5.14
-  - ANSIBLE_VERSION=2.5.13
-  - ANSIBLE_VERSION=2.5.12
-  - ANSIBLE_VERSION=2.5.11
-  - ANSIBLE_VERSION=2.5.10
-  - ANSIBLE_VERSION=2.5.9
-  - ANSIBLE_VERSION=2.5.8
-  - ANSIBLE_VERSION=2.5.7
-  - ANSIBLE_VERSION=2.5.6
-  - ANSIBLE_VERSION=2.5.5
-  - ANSIBLE_VERSION=2.5.4
-  - ANSIBLE_VERSION=2.5.3
-  - ANSIBLE_VERSION=2.5.2
-  - ANSIBLE_VERSION=2.5.1
-  - ANSIBLE_VERSION=2.5.0
```

Comment:
Any reason these are removed?

Reply:
This versions are no longer maintained by Ansible. The further check for Python 3 was doubling the builds. So I thought, don't stress Travis to much.